docs/cloud-native-security/session-view.asciidoc

```diff
@@ -92,6 +92,11 @@ Session View includes additional badges not pictured above:
 //[role="screenshot"]
 //image::images/session-view-script-button.png[The Script button]
 
+* The alert badge for multiple alerts. If a single event causes alerts of multiple types, multiple icons appear on the alert badge (a gear, a page, and a globe for `process`, `file`, and `network` alerts, respectively):
++
+[role="screenshot"]
+image::images/session-view-alert-types-badge.png[The alert badge for a command with all three alert types,width=55%,height=55%]
```

Comment:
Maybe scale down the image more? This looks too big on the preview. I played around on my machine and 35% seems on par with the other images below this one.
```suggestion
image::images/session-view-alert-types-badge.png[The alert badge for a command with all three alert types,width=35%,height=35%]
```